.ci/generate-snapshots.sh

```diff
@@ -68,6 +73,17 @@ for branch in $(jq -r '.branches | .[]' branches.json); do
   fi
 done
 
+echo ">> No more than 2 versions for the same minor is needed"
+if [ "$(jq -r '.branches | map(select(. | startswith("8."))) | length' branches.json)" == "3" ] ; then
```

Comment:
This is the new logic, as today:

```json
{
  "branches": [
    "7.17",
    "8.6",
    "8.7",
    "8.8",
    "main"
  ]
}

```


but `8.6` is not an active release branch, hence it should not be in the list of active branches